README.md

```diff
@@ -10,12 +10,34 @@ Install the package and all of its peer dependencies:
 npx install-peerdeps --dev @open-turo/eslint-config-typescript
 ```
 
-Then in your `.eslintrc` file add:
+Then in your `.eslintrc` file extend from one of the two configurations included in this package:
+
+1. The default, recommended for most Turo front-end projects
+2. The strict, that was being used for some node projects
```

Comment:
Should we still reverse the logic and keep the strict the default (with some required updates for that to be possible) and have a relaxed preset for legacy projects?

Reply:
> This is confusing. Why shouldn't be using the strict mode in any non legacy project?

This issue goes back to the tension I see between node and web projects.
The `strict` mode includes stuff that is node specific and/or is too rigid for our existing codebase (e.g. `unicorn`), meaning that we won't be able to use it for our current efforts on monorepo standards.
That's why we discussed creating a "recommended" eslint config that works for our front-end repositories.
It felt that our front-end projects are the main consumers of this `eslint` config, and in that context it made more sense to have that as the default

Reply:
I see. Outside of existing projects, where bringing up to these standards would be complex, what issues do you see in having strong rules like unicorn in new projects? Or is it really too strict?

Reply:
In my mind I'm thinking of existing projects as legacy and we should try to move them to the new world eventually and thus the new preset, to support them until we can afford to do that.

Reply:
> to support them until we can afford to do that

From my perspective, all the FE repositories, including the new ones created as part of extractions from the monorepo, will probably never use the `strict` preset, as it is currently configured.

If we don't feel comfortable having the front-end-compatible preset as the default, I can revert it; but it will force us to pick the right name for that preset. Maybe `stable`? And it will also mean we will have to change all the front-end packages `.eslintrc` to point to that other file.

I can do that. CC'ing @Toolo for awareness on this

Reply:
> will probably never use the strict preset, as it is currently configured.

What are the things / rules that are blocking this? If it's just unicorn, then let's see how we can relax it a bit

I think ideally achieving a place where both worlds meet would be great?

Reply:
Update:

We ended up keeping the original preset as the default and added a `legacy` one based on our front-end config.

The README has been updated accordingly in: https://github.com/open-turo/eslint-config-typescript/pull/118/commits/446ea9d4fb034282fb489bca9992c8c5533a439e